### ISSUE TITLE
sym.entry.scalaSig.hasEntry(idx) check in metacp

### ISSUE THREAD

**Issue by xeno-by** [scalameta/scalameta#1313]
This check (https://github.com/scalameta/scalameta/blob/49fde66004b997df8140c722e7637a50e32115df/semanticdb/metacp/src/main/scala/scala/meta/internal/metacp/Main.scala#L172-L180) was introduced as a stop-gap measure to unblock large-scale metacp tests. We should get back to it to see why it's needed and whether it's a principled enough fix.

**Comment by xeno-by** [comment]
This is no longer relevant in the latest master.
